core/domain/value_generators_domain.py

```diff
@@ -72,16 +79,15 @@ def generate_value(self, *args, **kwargs):
 
 class Registry:
     """Maintains a registry of all the value generators.
-
     Attributes:
         value_generators_dict: dict(str : BaseValueGenerator). Dictionary
             mapping value generator class names to their classes.
     """
 
-    value_generators_dict = {}
+    value_generators_dict: Dict[str, BaseValueGenerator] = {}
```

Comment:
This is not a list of generator instances. This is a list of generator classes. You will need to use `Type[]` here, see https://mypy.readthedocs.io/en/stable/kinds_of_types.html#the-type-of-class-objects

Reply:
Done.